docs/plugins/utils.md

```diff
@@ -324,13 +324,26 @@ ya.preview_widget(opts, {
 
 ### `sync(fn)` {#ya.sync}
 
+Make a function synchronous.
+
 See [Async context](/docs/plugins/overview#async-context).
 
 | In/Out | Type                 |
 | ------ | -------------------- |
 | `fn`   | `fun(...: any): any` |
 | Return | `fun(...: any): any` |
 
+### `async(fn)` {#ya.async}
+
+Execute a function in an asynchronous context while in a synchronous context and return its value.
```

Comment:
```suggestion
:::warning
This API is highly experimental at the moment, and its behavior may change in the future.
:::

Run a function asynchronously on the main thread.

`fn` should contain only async I/O operations, i.g., calls to other async APIs, and should not include any sync I/O, or blocking tasks, such as Lua's `io.open()`, `os.system()`.

`fn` runs in an asynchronous context but can access any [Sendable values](/docs/plugins/overview#sendable) from the outer synchronous context, for example:

```lua
--- @sync entry
local function entry()
  local cwd = cx.active.current.cwd
  ya.async(function ()
    ya.dbg(cwd)  -- `cwd` is a Url, which is sendable
  end)
end

return { entry }
```
```